tools/docker-utils.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const tempy = require('tempy');
+const execa = require('execa');
+const { getUnleashServerVersion } = require('./unleash-server-utils');
+
+const tempDirPath = tempy.directory();
+
+/** @type {string} */
+const dockerfileTemplate = fs.readFileSync(
+  path.resolve(__dirname, '..', 'Dockerfile'),
+  { encoding: 'utf8' },
```

Comment:
```suggestion
  path.resolve(__dirname, '../Dockerfile'),
  'utf8',
```

Reply:
1. does `../Dockerfile` work on Windows?
2. I usually prefer using options objects to name the options. Is there any big reason to prefer passing just the string (even if both work)? https://nodejs.org/api/fs.html#fs_fspromises_readfile_path_options seems to prefer passing an options object, but that may just be my bias towards named parameters